src/spell/modules/spell/spell/src/kv/collection.rs

```diff
@@ -85,6 +92,34 @@ pub fn list_get_strings(key: &str) -> StringListValue {
     result.into()
 }
 
+#[marine]
+/// Remove a value from a list of strings. If the value is in several places, remove all of them.
+/// `list_order` is used to preserve order of elements in a list, not the actual index.
+pub fn list_remove_value(key: &str, value: &str) -> UnitValue {
+    let conn = db();
+    let result: eyre::Result<()> = try {
+        let mut statement = conn.prepare(
+            r#"
+            DELETE FROM kv
+                WHERE key = ?
+                  AND string = ?
+                  AND list_order != -1
```

Comment:
What does -1 mean? Can you leave a comment on that plz?

Reply:
It's a default value for `list_order` when the entry isn't a list. I'll leave a comment.